packages/e2e/src/helper/createTestEnv.ts

```diff
@@ -27,8 +27,8 @@ export const CONFIG = {
     },
   },
   LIVE: {
-    nativeFundingAmount: '0.1',
-    ledgerDepositAmount: '10',
+    nativeFundingAmount: '0.8',
+    ledgerDepositAmount: '12',
```

Comment:
Bumped because of failing ci E2E tests running out of funds, could be reverted after @glitch003 reduces fees on naga-test